2024/day02/solution.md

```
@@ -0,0 +1,88 @@
+
+
+### Directoy commands
+- ```pwd``` --> print work directory. Gives the present working directory.
+
+### Sample output image : c:\Users\amita\Pictures\Screenshots\Screenshot 2024-10-10 175934.png
+
+- ```cd path_to_directory``` --> change directory to the provided path
+
+### Sample output image : c:\Users\amita\Pictures\Screenshots\Screenshot 2024-10-10 180007.png
+
+- ```cd ~ ``` or just  ```cd ``` --> change directory to the home directory
```

Markdownlint

Spaces inside code span elements

(MD038, no-space-in-code)